Libraries/LibWeb/DOM/Node.cpp

```diff
@@ -2362,13 +2362,16 @@ ErrorOr<String> Node::name_or_description(NameOrDescription target, Document con
                                 }
                             }
                         } else if (role == ARIA::Role::spinbutton || role == ARIA::Role::slider) {
+                            auto aria_data = MUST(Web::ARIA::AriaData::build_data(element));
```

Comment:
Instead of constructing the entire ARIA data, which doesn't seem cheap, can we just user the getters on the element itself?
```c++
auto aria_value_now = element.aria_value_now();
auto aria_value_text = element.aria_value_text();
```

Reply:
Thanks much, yup — that’s clearly better. Made it so